wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1942,3 +1945,35 @@ class FloatKeyViolation(ASTViolation):
 
     error_template = 'Found float used as a key'
     code = 449
+
+
+@final
+class ProtectedModuleMemberViolation(ASTViolation):
```

Comment:
I am not sure if this name is clear and meaningful enough, if you have some suggestions please let me know

Reply:
It is fine! Thanks!